instruments/tests/test_base_instrument.py

```diff
@@ -141,17 +141,15 @@ def fake_comports():
     """
     Generate a fake list of comports to compare against.
     """
-    fake_device = ListPortInfo()
+    fake_device = ListPortInfo(device='COM1')
     fake_device.vid = 0
     fake_device.pid = 1000
     fake_device.serial_number = 'a1'
-    fake_device.device = 'COM1'
```

Comment:
Device must now be initialized with the class. See explanation in #298. This is also compatible with pyserial3.4.

3 more identical changes throughout mocked classes.